### ISSUE TITLE
Whitelist "blob" and "data" schemes for Workers

### ISSUE THREAD

**Issue by shanewholloway** [denoland/deno#5683]
```javascript
const blob_src = new Blob(['console.log("Worker from BLOB")'])

new Worker(URL.createObjectURL(blob_src), {type:'module'})
```

On `deno 1.0.1  v8 8.4.300  typescript 3.9.2` results in the following unsupported scheme error:

```
error: Uncaught Error: Unsupported scheme "blob" for module "blob:http://deno-opaque-origin/953cec0c-2344-4c5b-89ee-c9e7b4de975e". Supported schemes: [
    "http",
    "https",
    "file",
]


```

Workers from Blob URLs is expected to work from what I see in #2726 and #2729.

Can the `"blob"` and `"data"` schemes be whitelisted for Deno Workers?

**Comment by EliteScientist** [comment]
This would be nice. I'm working on a project where I need to dynamically spawn threads. I've attempted to use this method and received the same error. I attempted to make a worker that allows me to pass the model in and use dynamic imports to load the code and still get the same error. 

The only other option I see is to create an internal HTTP Server that will serve the worker code. (I'd prefer not to do this because its bulky) but there is a huge need to be able to create Workers from code for our product.

Its not only whitelisting the blob and data urls but URL.createObjectURL was removed in 1.4.1 (not sure why as its in the spec).

I'm using 1.4.5 right now.
